app/models/ems_folder.rb

```diff
@@ -118,6 +118,12 @@ def parent_datacenter
     detect_ancestor(:of_type => "EmsFolder") { |a| a.kind_of?(Datacenter) }
   end
 
+  # Indicates if the folder is able to have child VMs
+  def vm_folder?
```

Comment:
You might created your branch from the wrong one :wink:

Reply:
Ha yeah just noticed that, fixed